### ISSUE TITLE
Section 508 compliance issues

### ISSUE THREAD

**Issue by hermidalc** [CBIIT/nci-ctd2-dashboard#156]
*Originally reported by*: @armish

From Cliff:

I quickly glance over it and noticed that you have also made lots of progress on making the site section 508 compliant -  noticeably that you have ‘alt’ tag on all images in the html which is great.     I found only the following based on my visual inspection:
1.       The word Overview in the header CTD2 Dashboard Overview will failed the color contrast test ( it has a foreground color #999999 on a white background #ffffff)

2.       On the Browse Targets page,  the paragraph “Below is a list of targets…” will fail the color contrast test as well ( it has a foreground color #c09853 on a background #fcf8e3).

3.       Same as above, on the Browse compounds page,  the paragraph “Below is a list of compounds…” will fail the color contrast test as well.

All these could be addressed easily by increasing the color contrast, you can verify the pick using this checklist < http://webaim.org/standards/508/checklist> .

 
   The other one is the bullets that are used to indicates page 2, 3, 4 of the stores. Since they have the same presentation(same display character), it would be hard for the screen reader to tell which bullet visually impaired users intend to click. Have you consider using just numbers like 1, 2, 3, 4 ? by the way, the color contract of the current page will fail the contrast test  (same color as #1 above).

**Comment by hermidalc** [comment]
*Original comment by*: @armish

These are addressed as of rev 76a4593.

The stories bullet points are left as they are, as in the past it was decided that numbering these items would cause a false sense of priority across stories.
